src/app/view/perspectives/post-it/document/post-it-document.component.html

```diff
@@ -29,6 +32,7 @@
       <li *ngFor="let attributePair of attributePairs; let row = index">
         <span id="{{ perspectiveId }}{{ postItModel.index }}[0, {{ row }}]" class="attribute"
               tabindex="-1"
+              title="{{ attributePair.attribute }}"
```

Comment:
Why not simply using property binding (`[title]="attributePair.attribute"`) instead of interpolation?